assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+## Project documentation 
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required) |
+
+### V1
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           |     |     | ✅  |     |     |
+| Content maintainability    |     | ✅  |     |     |     |
+| Content creation processes |     |     | ✅  |     |     |
+
+### V2
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           | ✅  |     |     |     |     |
+| Content maintainability    | ✅  |     |     |     |     |
+| Content creation processes | ✅  |     |     |     |     |
+
+### Comments
+
+There are two versions of notary that need to be supported with documentation.
+- V1 is housed with [The Update Framework](https://github.com/theupdateframework/notary), and the docs are included as a set of markdown files in the [/docs folder](https://github.com/theupdateframework/notary/tree/master/docs)
+- V2 is housed with The Notary Project (https://github.com/notaryproject/notaryproject) 
+
+**Information Architecture**: 
+- As there is no website as yet, and development is still ongoing, V2's Project Documentation is all contained within markdown files in the notaryproject repo.
+
+**New user content**:
+- What is there does a good job of explaining the project, but doesn't provide many docs focused on new users.
+
+**Content maintainability**:
+- The docs currently aren't versioned or localized, and is depending on GitHub for search.
+
+**Content creation processes**:
+- There are links to community discussions, but as there is no website yet there is no space to point to regarding docs creation.
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The notaryproject.dev site should primarily focus on the V2 work, but V1 needs to be represented as well.
+- V1 content seems quite good, but looks like it was developed for a site that no longer uses it.
+- For MVP, I'd suggest focusing on V2 content, and linking out to the existing V1 documentation in the location that is currently rather than incorporating it right off the bat.
```

Comment:
As a general comment: while I understand that this work is being done in advance of doing a website for the Notary project, your recommendations are so strongly geared _only_ towards what needs to be done that it's _only_ useful for you. 

The purpose of this document is to give _the project_ enough information that they can action on things on their own. Using phrasing like "For MVP", and describing recommendations in vague terms like this doesn't help them achieve that aim.

Reply:
I agree, the draft you reviewed was written with myself as the audience. Thanks for this feedback, I'm keeping it in mind during my revisions.